src/manual/media-recorder/js/main.js

```diff
@@ -0,0 +1,441 @@
+/*
+*  Copyright (c) 2016 The WebRTC project authors. All Rights Reserved.
+*
+*  Use of this source code is governed by a BSD-style license
+*  that can be found in the LICENSE file in the root of the source
+*  tree.
+*/
+
```

Comment:
Add `/* exported varNames */` for all the variables called upon elsewhere (window.onerror(lineno..), resolve, reject, descrption etc).

Add `/* globals varNames */` for all the variables defined elsewhere (MediaRecorder, Promise)